### DISCUSSION TITLE
スマホで記事の更新ボタンが押せない（表示されない）

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#220] [要望・アイデア・不具合]
Qiitaの記事を、スマホで修正しようとした際に、更新ボタンが表示されないように思います。

以前は、一番下までスクロールすれば押せたと思うのですが、今はスクロールしても更新ボタンが表示されず、更新できないです。

* android 12
* chrome 101.0.4951.61

（単純に、私の操作ミスならごめんなさい）

できれば、一番下までスクロースしないでも更新ボタンが常時表示されているようにしてもらえるとありがたいです。

**Comment by tanaka-asami415** [comment]
@yoshi389111 
ご不便をおかけして申し訳ありません🙇
こちら現在修正対応中です！修正が完了しましたら、こちらでもご報告させていただきます🙇

**Reply by tanaka-asami415** [reply]
@yoshi389111 
お待たせして申し訳ありません！こちら修正完了しております🙇
